sdk/core/core-client/src/serializer.ts

```diff
@@ -154,8 +154,6 @@ class SerializerImpl implements Serializer {
     if (object === undefined || object === null) {
       payload = object;
     } else {
-      // Validate Constraints if any
-      this.validateConstraints(mapper, object, objectName);
```

Comment:
I get the rationale for not removing the method, but can we mark it `@deprecated`?

Reply:
Done